VaticanBAV/et/BAVet118.xml

```diff
@@ -201,11 +188,85 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                               <layout columns="2" writtenLines="22"/>
                           </layoutDesc>
                       </objectDesc>
+                      <handDesc>
+                          <handNote xml:id="h1" script="Ethiopic">
+                              <seg type="script"/>
+                              <seg type="ink">Black, red</seg>
+                              <date notBefore="1850" notAfter="1920"/>
+                              <desc>
+                              </desc>
+                              <seg type="rubrication">
+                                  Titles, both in Ethiopic and in Greek.
+                              </seg>
+                          </handNote>
+                      </handDesc>
+                      
+                      <additions>
+                          <list>
+                              <item xml:id="a1">
+                                  <locus from="76v" to="77r"/>
+                                  <desc type="GuestText"><title ref="LIT7138PrLover">Prayer of the lover</title></desc>
+                                  <note>
+                                      Ethiopic prayer, entitled  <q xml:lang="grc">περη αγαπης κὲ εφτηχία</q>.
+                                      It begins as follows: <q xml:lang="gez">
+                                          በስመ፡ አብ፡ <gap reason="ellipsis" resp="PRS4805Grebaut PRS9530Tisseran"/> <sic>ጸሎተ፡</sic> <sic>ምስተ<surplus>፡</surplus>ፋቅር፨</sic> 
+                                          አስማተ፡ ንዋይ፡ ምስሓበ፡ ንዋይ፨ አስተፋቅር፡ ወአስተዋድድ፡ ወአስተ<surplus>፡</surplus>ሳልም፨
+                                      </q>, and concludes as follows: 
+                                      <q xml:lang="gez">ኵሎሙ፡ ሰብአ፡ ሃገር፡ <sic>ወሉደ፡</sic> አዳም፡ ወሔዋን፡ ይሰግዱ፡ ለገብርከ፡ እገሌ፨
+                                      </q>.
+                                      The use of this formula is explained on <locus target="#77rb"/> in Greek and Gǝʿǝz-Amharic as follows: 
+                                      <q xml:lang="grc">τά άνωθεν τά γράφης σε πετζή γενας μπράια μεγάλη</q>
+                                      <q xml:lang="am">የላይኛውን፡ ቃል፡ በብርሃና፡ ጸሀፎ፡ <surplus resp="PRS4805Grebaut PRS9530Tisseran">ለ</surplus>ትጽህፎ፡ በወርቅ፡ ቀለም፡ 
+                                          በሰባ<supplied reason="omitted">ዕ</supplied>ቱ፡ ቀለማት፡ ፪<del rend="expunctuated">ገ</del>ጊዜ፡</q>
+                                  </note>
+                              </item>
+                              
+                              <item xml:id="e1">
+                                  <note>
+                                      Many spaces left blank were filled in with Greek remedies and short texts.
```

Comment:
What do you mean with 'remedies'? Are they 'corrections', 'medecine', 'prayers'?